### PR TITLE
SLM in public docs 

### PR DESCRIPTION
**Pull request by urbiz-nr**
Moving SLM docs to public site. See DOC-7558 for more details.

1st commit:
- Updated nav file
- Created folder for SLM docs, with intro doc and images
- Failed to add tile to docs.newrelic.com

### CODE REVIEW

**Comment by urbiz-nr** [comment]
@x8a - here's the PR I was talking about. See if you can figure out how to edit the landing page. Thanks so much!

**Comment by urbiz-nr** [comment]
Tried locally - all looks good, images render, links work.

PENDING:
- Tile in docs.newrelic.com
- Add category in left nav

**Review by x8a — APPROVED** [review]
Happy to see these docs on the public site 💃🏻💃🏻💃🏻

File: src/i18n/translations/en/translation.json [hunk 1]
```diff
@@ -112,7 +112,11 @@
       "t11": {
         "title": "Network Performance Monitoring",
         "description": "Monitor your network devices."
-      }
+      },
+      "t12": {
+        "title": "Service Levels management",
+        "description": "Configure and consume SLIs and SLOs."
+      },
```
Comment (rudouglas) [inline src/i18n/translations/en/translation.json:119]:
```suggestion
      }
```

**Pull request merged**